homeassistant/components/ecovacs/vacuum.py

```diff
@@ -1,6 +1,17 @@
 """Support for Ecovacs Ecovacs Vaccums."""
 import logging
 
+from sucks import (
+    FAN_SPEED_HIGH,
+    FAN_SPEED_NORMAL,
+    Charge,
+    Clean,
+    PlaySound,
+    Spot,
+    Stop,
```

Comment:
It feels a bit risky to have some of these imports have such a generic imported name (like `Stop()`) at a higher scope than they were before. It certainly reduces readability in this case, as it won't be obvious what the `Stop()` method comes from when it's used in one method later on.